Delist 'Start Building RESTful Microservices using Akka HTTP with Scala'

psilospore opened the pull request:
I checked out this book after seeing it on the docs and found it to be poorly written. I don't think this book should be listed as a resource as it's not very helpful for readers of any level. I'm not sure if there are standards for what makes a resource good enough to be listed but if there were any I don't believe this book would meet the criteria.

psilospore committed:
Delist 'Start Building RESTful Microservices using Akka HTTP with Scala'

I don't believe this book was a particularly good resource. This shouldn't be listed as a resource.

lightbend-cla-validator commented:
Hi @psilospore,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

ktoso reviewed (commented):
Hi and thanks for bringing this up @psilospore.
I got the "book" and agree it is of not so great quality, rather than a book one could have called it a pretty long blog post really. It does not seem to be really wrong so I don't want to de-list it since this page is also to serve our community to get together and be happy about the work they've done.

What I would instead do here is the following (and please amend the PR if you feel comfortable with the suggestion, would love to merge it then):

- put the mentioned book at the bottom of the list.
- put `  * [Reactive Design Patterns](https://www.reactivedesignpatterns.com/), by Roland Kuhn with Jamie Allen and Brian Hanafee, Manning Publications Co., ISBN 9781617291807, Feb 2017 ` and `* [Akka in Action](http://www.lightbend.com/resources/e-book/akka-in-action), by Raymond Roestenburg and Rob Bakker, Manning Publications Co., ISBN: 9781617291012, September 2016` to the top of the list and make them be "Recommended reads: " the others being "Other reads about Akka and the Actor model:"

WDYT?

psilospore committed:
Books are now seperated into a recommended section and others

Books are now seperated into a recommended section and others.
Start Building RESTful Microservices using Akka HTTP with Scala is placed on the bottom.

psilospore committed:
Merge branch 'master' into patch-1

renatocaval commented on akka-docs/src/main/paradox/additional/books.md:
```diff
@@ -1,10 +1,12 @@
+
  * [Mastering Akka](https://www.packtpub.com/application-development/mastering-akka), by Christian Baxter, PACKT Publishing, ISBN: 9781786465023, October 2016
  * [Learning Akka](https://www.packtpub.com/application-development/learning-akka), by Jason Goodwin, PACKT Publishing, ISBN: 9781784393007, December 2015
  * [Akka in Action](http://www.lightbend.com/resources/e-book/akka-in-action), by Raymond Roestenburg and Rob Bakker, Manning Publications Co., ISBN: 9781617291012, September 2016
```
This is appearing twice. In the recommended reads and here.

ktoso committed:
Update books.md

ktoso reviewed (approved):
LGTM thanks

ktoso commented:
Removed the duplicate
